tests/console/check_package_version.pm

```diff
@@ -24,7 +24,7 @@ use testapi;
 use utils qw(systemctl zypper_call);
 use Mojo::Util 'trim';
 
-my %package = (
+my %package_12sp5 = (
```

Comment:
maybe we can remove 12sp5 part since project already finished.

Reply:
If you think it is no value to keep it. I can remove them.